test_tictactoe.py

```diff
@@ -9,7 +9,7 @@ def test_new_game(self):
         self.fail("not implemented")
 
     def test_get_starting_player(self):
-        self.assertIsInstance(get_starting_player(), bool)
+        self.assertIsInstance(get_starting_player(human), True)
```

Comment:
`get_starting_player()` doesn't take parameters. More sophisticated testing is needed if the return value is random.